src/pages/Product/index.js

```diff
@@ -1,9 +1,168 @@
+        <div className={styles.info}>
+          <div className={styles.price}>${price}</div>
+          <div className={styles.title}>{title}</div>
+          <div className={styles.description}>{shortDescription}</div>
+          <div className={styles.qty}>
+            <p className={styles.qtyTitle}>Qty</p>
+            <div className={styles.qtyCount}>
+              <input
+                className={styles.input}
+                onChange={event =>
+                  enterQtyProductInDetails(parseInt(event.target.value))
+                }
+                type="text"
+                placeholder="Input field"
+                value={qtyProduct}
+              />
+              <div className={styles.qtyControl}>
+                <div
+                  onClick={() => changeQtyProduct(CHANGE_QTY_PRODUCT_UP)}
+                  className={styles.qtyControlTop}
+                >
+                  <Arrow />
+                </div>
+                <div
+                  onClick={() => changeQtyProduct(CHANGE_QTY_PRODUCT_DOWN)}
+                  className={styles.qtyControlBottom}
+                >
+                  <Arrow />
+                </div>
+              </div>
+            </div>
+          </div>
+          <div
+            className={styles.btn}
+            onClick={() => btnStatus(idProduct, qtyProduct, addToCart)}
+          >
+            Add to cart
+          </div>
+        </div>
+      </div>
+    );
   }
 }
 
-export default Product
+const Slider = ({ listImg, activeImg, chooseImg, loader }) => {
+  if (loader) {
+    return (
+      <div className={styles.loading}>
+        <Loading />
+      </div>
+    );
+  } else {
+    return (
+      <div className={styles.slider}>
+        <div className={styles.slide}>
+          <div className={`${styles.inner} ${styles.correlationHeight}`}>
+            <div className={styles.content}>
+              <img
+                className={`${styles.img} ${styles.imgMain}`}
+                src={activeImg.large}
+                alt="product"
+              />
+            </div>
+          </div>
+        </div>
+        <div className={styles.list}>
+          {listImg.map(({ id, small, large }, index) => (
+            <div
+              key={id}
+              className={
+                activeImg.large === large
+                  ? `${styles.item} ${styles.itemActive}`
+                  : `${styles.item}`
+              }
+              onClick={() => chooseImg(index)}
+            >
+              <div className={`${styles.inner} ${styles.correlationHeight}`}>
+                <div className={styles.content}>
+                  <img className={styles.img} src={small} alt="product" />
+                </div>
+              </div>
+            </div>
+          ))}
+        </div>
+      </div>
+    );
+  }
+};
+
+const btnStatus = (idProduct, qtyProduct, addToCart) => {
+  if (qtyProduct.length === 0) {
+    return null;
+  } else return addToCart(idProduct);
```

Comment:
if you use return inside `if(){}` construction 
`else` is useless just return function

```
if (qtyProduct.length === 0) {
    return null;
  } 
 return addToCart(idProduct);
```